poly-commitment/src/commitment.rs

```diff
@@ -456,10 +460,17 @@ pub struct Evaluation<G>
 where
     G: AffineCurve,
 {
-    /// The commitment of the polynomial being evaluated
+    /// The commitment of the polynomial being evaluated.
+    /// Note that PolyComm contains a vector of commitments, which handles the
```

Comment:
Thanks for this, it is the thing I always forget about and always keep asking myself why `[0]`...